internal/lifecycle.go

```diff
@@ -83,7 +86,7 @@ func (ls *lifecycleService) fireLifecycleEvent(newState string) {
 			listener.(core.LifecycleListener).LifecycleStateChanged(newState)
 		}
 	}
-	log.Println("New State : ", newState)
+	ls.logger.Info("New State : ", newState)
```

Comment:
See the same method in Java and look at the content of the message, this content is different. It is better all clients logs the same information so that it is easier for the user to trace what is going on.

Reply:
I have changed it as `ls.logger.Info("HazelcastClient", ClientVersion, "is", newState)`. We dont have buildInfo in Go client like Java. Is that okay?